### ISSUE TITLE
Veil3 fails to setup in Wine

### ISSUE THREAD

**Issue by PoSHMagiC0de** [Veil-Framework/Veil#132]
## Veil version
3.1.1

## OS Used - all info (architecture, linux flavor, etc)
ParrotOS 3.0 Debian 9
Linux parrot 4.13.0-parrot4-amd64 #1 SMP Parrot 4.13.4-2parrot0 (2017-10-17) x86_64 GNU/Linux
## How did you install Veil? (Apt, Clone from Github, etc.)
Noticed issue first with repo with apt and updated to veil3.  After that tried a new virtual machine to test OS and still same issue.  Then tried downloading from git repo and then setup.sh -c script as sudo and directly as root and still same error.

## Did you run the setup script?
Yes.

## Pastebin link to error you are encountering (include console actions you took prior to error)
Through Git repo and after apt install I went into the veil/setup folder of either one and ran:

sudo ./setup.sh -c

https://pastebin.com/VJk1N3ur

## Expected behavior
Installation of packages to complete and launching of veil to do stuff.

## Any additional info you want to tell me

It took Parrot some time to allow Veil3 package into repo from Kali.  It was allowed recently because I was on the old version 2 Veil last week and was running fine.  After the update to 3 where I began having issues.  I notice one of the errors are complaining about the $DISPLAY variable being null.  This is not null.  I can run both as sudo and normal user wine apps with no null display.  It just seems to be when running Veil.  I also notice a new app called firejail that I do not remember seeing when running.

**Comment by PoSHMagiC0de** [comment]
Disregard.  I am a shitty troubleshooter.  Firejail was the new program added, firejail was the culprit after research.  I guess they added it for protect but breaks stuff in wine due to restricting access, even when I tried to modify the profiles.  I just uninstalled it.  Works fine now.

**Comment by PoSHMagiC0de** [comment]
Spoke too soon.  After removing firejail and getting a good install I tried to run veil and it works but both ordinance and evasion return back to main menu with nothing when I try and list or use them.

I have done the update.sh as sudo from the config folder to point to metasploit folder as:
/usr/share/metasploit-framework/

like in the previous version when you show no payload but no cigars.

**Comment by ChrisTruncer** [comment]
Can you show a new pastebin of the error?

**Comment by PoSHMagiC0de** [comment]
So, when I had time I went through the new code for veil to see if something is breaking or I am using it wrong.  I learned the new commands and now it works fine.  User error.  "List" needs to be used with parameters.  Then I realized main website was updated with new commands for 3.0.  So, one was the issue of a new app installed in my distro I didn't expect called "Firejail".  The other is I dived in like i was using old Veil instruct of RTFM.
